Tools/jsdoc/cesium_template/static/styles/cesium-style.css

```diff
@@ -0,0 +1,3 @@
+#ClassList > li {
```

Comment:
This isn't the only customization we've done to jsDoc.  jsdoc-default.css contains Cesium's rules.

Reply:
I wasn't sure whether we should be modifying that, but I can just move that there then.

Reply:
"Should be" is sadly a different question from "have we been doing it that way from the start".  Not saying this was the best or only way, but, Cesium has customized more than just the CSS, there are custom JS-based changes as well.  The addition of anchor links, the color swatch documentation, the popout windows from Sandcastle, and others may all have custom JS.

Reply:
We really need to go back to stock jsdoc and update everything the right way at some point.

Reply:
To be clear, everything in `cesium_template` is "owned" by Cesium and can be modified at will for however we want the output to look. (hence the name).  As far as I can tell from a quick check on the JSDoc website, ~6 years later, forking and modifying the template is still the plugin point to use to get the output we want.

Reply:
Yes.  Apologies if I implied we did it wrong, this was the established practice then too.  There's no "fixing" it without adding a lot more plugin API points to JSDoc.